packages/app/lib/i18n/en.i18n.json

```diff
@@ -33,5 +33,10 @@
   "street": "Street",
   "zip": "Zip",
   "city": "City",
-  "open_hours": "Open hours"
+  "open_hours": "Open hours",
+  "appeals": {
+    "draft": "<strong>Draft</strong>: This appeal is not yet made.",
```

Comment:
Ok thanks! Please take over there – as you're the expert for English :wink: 
im already on the next PR